test/simulation/actions.go

```diff
@@ -70,12 +70,18 @@ func (*actions) connect(ctx context.Context, assert *require.Assertions, srcNode
 
 func (*actions) placeOrderAndBroadcast(ctx context.Context, assert *require.Assertions, srcNode, destNode *xudtest.HarnessNode,
 	req *xudrpc.PlaceOrderRequest) *xudrpc.Order {
+	// Subscribe to added orders on destNode
+	destNodeAddedOrderChan := subscribeAddedOrders(ctx, destNode)
+
 	// 	Fetch nodes current order book state.
 	prevSrcNodeCount, prevDestNodeCount, err := getOrdersCount(ctx, srcNode, destNode)
 	assert.NoError(err)
 
-	// Subscribe to added orders on destNode
-	destNodeAddedOrderChan := subscribeAddedOrders(ctx, destNode)
+	// Ensure that destNode is connected to srcNode
+	resListPeers, err := destNode.Client.ListPeers(context.Background(), &xudrpc.ListPeersRequest{})
+	assert.NoError(err)
+	assert.Len(resListPeers.Peers, 1)
```

Comment:
Node might have more peers connected. Just need to check that this peer is connected, not that it's the only one.

Reply:
Good point.